File: app/src/main/java/com/yesmeal/yesmealnotes/fragments/navigation/ManageStaffs.java
package com.yesmeal.yesmealnotes.fragments.navigation;

import android.app.Fragment;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ExpandableListAdapter;
import android.widget.ExpandableListView;

import com.yesmeal.yesmealnotes.ymutils.StaffExpandableListAdapter;

import com.yesmeal.yesmealnotes.R;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;

public class ManageStaffs extends Fragment {
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.frag_manage_staffs,container,false);
    }

    @Override
    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {

        ExpandableListAdapter expandableListAdapter;
        List<String> expandableListTitle;
        HashMap<String, List<String>> expandableListDetail;
        ExpandableListView expandableListView  = view.findViewById(R.id.expandableListView);

        expandableListDetail = ExpandableListDataPump.getData();
        expandableListTitle = new ArrayList<String>(expandableListDetail.keySet());
        expandableListAdapter = new StaffExpandableListAdapter(getContext(), expandableListTitle, expandableListDetail);
        expandableListView.setAdapter(expandableListAdapter);


        int count = expandableListAdapter.getGroupCount();
        for ( int i = 0; i < count; i++ )
            expandableListView.expandGroup(i);



        expandableListView.setOnGroupClickListener(new ExpandableListView.OnGroupClickListener() {
            @Override
            public boolean onGroupClick(ExpandableListView expandableListView, View view, int i, long l) {
                return true;
            }
        });

    }
}

File: app/src/main/java/com/yesmeal/yesmealnotes/fragments/navigation/ManageStaffs.kt
package com.yesmeal.yesmealnotes.fragments.navigation

import android.app.Fragment
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ArrayAdapter
import com.yesmeal.yesmealnotes.R
import com.yesmeal.yesmealnotes.adapters.ParentZoneAdapter
import com.yesmeal.yesmealnotes.models.Zone
import kotlinx.android.synthetic.main.common_list_view.*

/**
 * Created by asnimansari on 03/02/18.
 */


class ManageStaffs:Fragment(){
    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        var db = MySqlHelper.getInstance(context)
        var zonelList = db.getAllZoneList()  as List<String>

        var arraAdapter = ParentZoneAdapter(context,android.R.layout.simple_spinner_item,zonelList)
        listView.adapter = arraAdapter


    }

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View {
        return inflater?.inflate(R.layout.common_list_view,container,false)!!
    }
}